src/test/rgw/test_rgw_lua.cc

```diff
@@ -280,6 +280,66 @@ TEST(TestRGWLua, URI)
   ASSERT_EQ(rc, 0);
 }
 
+TEST(TestRGWLua, RGWDebugLog2)
```

Comment:
please add tests to cover negative cases. especially:
* invalid format tring
* format strings with" escape squences, positional arguments, and format specs
* too many/not enough arguments
* argument that cannot be converted to string
* nil values either literal or from a variable

please also add:
please add a test to cover using a value from the request context itself. e.g. `RGWDebugLog2("Testing {}", Request.Object.Name)`
table values. e.g. `RGWDebugLog2("Testing {}", Request)